Kudu.Core/Environment.cs

```diff
@@ -1,13 +1,13 @@
-﻿using System;
+﻿using Kudu.Contracts.Settings;
```

Comment:
I believe we order but put all 'using System's at the top. It's one of the options in VS